systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -123,24 +129,29 @@ def add_rule(self, cidr):
                                     " -p %s " % rule['protocol'] +
                                     " -m %s " % rule['protocol'] +
                                     " --dport %s -j RETURN" % rnge])
+
+            logging.debug("Current ACL IP direction is ==> %s", self.direction)
             if self.direction == 'egress':
-                self.fw.append(["filter", "", " -A FW_OUTBOUND -j FIREWALL_EGRESS_RULES"])
+                self.fw.append(["filter", "", " -A FW_OUTBOUND -j FW_EGRESS_RULES"])
```

Comment:
No, I renamed to make it consistent with the default rules, which can be found in the iptables-router file. That file is installed with cloud early config then afterwards the rules are replaced.

In addition, the code was also refering to FW_OUTBOUND, not FIREWALL_OUTBOUND. The use of FW cs FIREWALL doens't look consistent. Now we have only FW.

I hope it's clear. I am at home today... A bit ill. So typing from my mobile

Cheers,
Wilder